tests/baselines/reference/autoAccessor6(target=esnext,usedefineforclassfields=false).js

```diff
@@ -0,0 +1,26 @@
+//// [autoAccessor6.ts]
+class C1 {
+    accessor a: any;
+}
+
+class C2 extends C1 {
+    a = 1;
+}
+
+class C3 extends C1 {
+    get a() { return super.a; }
+}
+
+
+//// [autoAccessor6.js]
+class C1 {
+}

```

Comment:
this emit is weird, but I *guess* correct?

Reply:
Actually, this is wrong. The field should have been included in the emit. I'll investigate.